src/state/reducers/rootReducer.js

```diff
@@ -0,0 +1,18 @@
+import initialState from '../store/initialState'
+const rootReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'SET_ANSWERS':
+      let newData = state.formData
+      newData[action.payload.key] = action.payload.answer
+      return { ...state, formData: newData }
```

Comment:
Could we format this so the objects are on new lines? :3

Reply:
done